ament_cmake_core/cmake/core/package_xml_2_cmake.py

```diff
@@ -73,6 +76,20 @@ def get_dependency_values(key, depends):
     return dependencies
 
 
+def escape_back_slash(line):
```

Comment:
Yeah, I was going to use `os.path.normpath`, but it does the opposite of what I want:

> On Windows, it converts forward slashes to backward slashes.

Maybe I should call it `replace_backslashes` or `replace_backslashes_with_forward_slashes`. Because that's really what it's doing. The result of this gets templated into CMake files, which unfortunately cannot parse files with backslashes in them. I'll update it to something appropriate.